woocommerce/woocommerce-ios
# [Analytics Hub] Update Analytics cards with real line chart data

Part of #8201
⚠️ Depends on #8231 ⚠️ 

## Description

Updates the Revenue and Orders cards in Analytics Hub with real line chart data.

## Changes

* Adds a helper to get the chart data for a particular type of stats total (total revenue, total orders, etc.). This parses the order stats to get the requested total from each stats interval.
* Updates the data bindings in the Analytics Hub view model to use real chart data.

## Testing

1. Launch the app.
2. Tap the "See More" button from the main dashboard.
3. See the Analytics view loads with real charts (reflecting actual store data) on the Revenue and Orders cards.

**Note:** The Analytics Hub currently gets the same interval unit and quantity (31 days of interval data) regardless of the selected analytics time period, so for now the line charts will look the same for every selected time period.

## Screenshots

<img src="https://user-images.githubusercontent.com/8658164/204298727-2e6b6646-7310-4dbc-9e32-028eda4e59b1.png" width="300px">


## Submitter Checklist

Update release notes:
